### ISSUE TITLE
Melt Deformer still applies Noise when Factor is 0

### ISSUE THREAD

**Issue by hybridherbst** [keenanwoodall/Deform#15]
Seems other Deformers (that I tried) follow the (useful) convention that Factor basically allows fading the Deformer in and out. Melt doesn't do that, but "jumps" on Factor 0 (Factor 0.0001 already applies full Noise).

![20190708-005947](https://user-images.githubusercontent.com/2693840/60774918-b686fe80-a11b-11e9-87ac-1bfc4f8bb861.gif)

**Comment by keenanwoodall** [comment]
Good find. The vertical noise's strength is not multiplied by the factor. It turns out the radial noise *is* multiplied by the factor. I changed the vertical noise offset so that it is first multiplied by the factor before being added to the vertices position. When playing with the factor to make sure it worked I discovered that multiplying by the factor is not a good solution. Since factor is not clamped the noise's strength will surpass its defined magnitudes when factor is greater than 1. The fix I will soon be pushing changes factor to be a slider between 0 and 1. The effect caused by going beyond a factor of 1 can be achieved by just increasing the radius. Factor = 2, Radius = 1 is the same as Factor = 1, Radius = 2. Clamping factor between 0 and 1 will let me use it to mask the noise while still allowing users to amplify the strength of the melt by increasing the radius.
